parsl/monitoring/logging_server.py

```diff
@@ -0,0 +1,42 @@
+import tornado.ioloop
+import tornado.web
+import json
+from parsl.monitoring.db_logger import get_db_logger
+
+
+class MainHandler(tornado.web.RequestHandler):
+    """ A handler for all requests/logs sent to the logging server."""
+    def initialize(self, db_logger_config_object):
+        """This function is called on every request which is not ideal but __init__ does not appear to work."""
+        self.db_logger_config_object = db_logger_config_object
+
+    def get(self):
+        """Defines responses to get requests for the / ending. Not used by Parsl but could be."""
+        self.write('Hello world - Parsl Logging Server')
+        self.flush()
+
+    def post(self):
+        """
+        Defines responses to post requests for the / ending. Receives logs from workers and main dfk in the body of the post request.
+        Should be log=json.dumps(info). Then writes this info to the database using the database handler.
+        This needs to be a quick function so that the server can accept other requests and may be a bottle neck for incoming logs.
+        """
+        arg = json.loads(self.get_body_argument('log'))
+        try:
```

Comment:
can this initialisation of `self.application.logger` happen in an initialisation function?